actionpack/lib/action_dispatch/journey/route.rb

```diff
@@ -10,12 +10,12 @@ class Route
       module VerbMatchers
         VERBS = %w{ DELETE GET HEAD OPTIONS LINK PATCH POST PUT TRACE UNLINK }
         VERBS.each do |v|
-          class_eval <<-eoc
-          class #{v}
-            def self.verb; name.split("::").last; end
-            def self.call(req); req.#{v.downcase}?; end
-          end
-          eoc
+          class_eval <<-EOC, __FILE__, __LINE__ + 1
```

Comment:
@matthewd is talking about the needless `end_eval` to `EOC` change. Just change that back.